app/controllers/api/generic_works_controller.rb

```diff
@@ -0,0 +1,98 @@
+module API
+  # Adds an endpoint that customizes the update action on nested works
+  class GenericWorksController < ApplicationController
```

Comment:
Why are we adding a GenericWorksController?  The future direction for sufia is to not use GenericWorks and use whatever work types the user wants to generate (like CurationConcerns).